main.py

```diff
@@ -0,0 +1,171 @@
+"""
+Weathermood: A vehicle for nostalgia
+"""
+
+import ui
+from classes import Weather, WeatherMood, build_weathermood_object, Location, Playlist
+from get_location import get_location
+from getWeatherForecast import get_weather_forecast
+from spotify_api import search_spotify_playlists
+from consolemenu import *
+from consolemenu.items import *
+from geopy.geocoders import Nominatim
+import geopy.geocoders
+import requests
+
+one = WeatherMood(None, 0, 1710555589, 'Moscow', 'Idaho', 46.7323875, -117.000165, 47.25, 4.61, '01n', 'Clear',
+                      53, 'Russian Cream',
+                      'https://mosaic.scdn.co/640/ab67616d0000b27314d91ebdd6d7e2931322cc1aab67616d0000b2734db62465d930f949ecd18c38ab67616d0000b273565821887210741234f504e4ab67616d0000b273f2a7ea53a152b5b3ac9549a8',
+                      'https://open.spotify.com/playlist/6s9bX6r6v6tPvpkB269GYo')
+two = WeatherMood(None, 1, 1710557186.747964, 'Park Rapids', ' Minnesota', 46.9221813, -95.0586322, 48.13,
+                      17.27, '04n', 'Clouds', 50, 'Clouds Radio',
+                      'https://seeded-session-images.scdn.co/v2/img/122/secondary/track/03v70ZBxmcPX3RWAZMzqaW/en',
+                      'https://open.spotify.com/playlist/37i9dQZF1E8QyZpgdVgaHA')
+three = WeatherMood(None, 0, 1710557300.794149, 'Holly', ' Michigan', 42.7919727, -83.6277255, 37.09, 5.68,
+                        '01n', 'Clear', 310, 'Timeless Nostalgic Hits',
+                        'https://image-cdn-ak.spotifycdn.com/image/ab67706c0000bebb40557fc596eb62a51bfedd05',
+                        'https://open.spotify.com/playlist/5wcoLsZw0Rp1rsB3ydxeJE')
+
+
+fake_db = [one, two, three]
+# validity checks
+
+
+
+
+
+
+
+
+def is_valid_us_city(city_name):
+    """Checks if a city name is a valid US city using multiple methods."""
+
+    # Method 1: Using a pre-built dataset (if available)
+    # with open("us_cities.txt", "r") as f:  # Replace with your city dataset
+    #     us_cities = set(f.read().splitlines())
+    # if city_name.lower() in us_cities:
+    #     return True
+
+    # Method 2: Using a geocoding service
+    geolocator = geopy.geocoders.Nominatim(user_agent="city_validation_script")
+    location = geolocator.geocode(city_name, country_codes="US")
+    if location and location.address.split(",")[-1].strip() == "United States":
+        return True
+    return False
+
+
+
+moods = [
+  "happy", "sad", "angry", "excited", "calm",
+  "frustrated", "joyful", "peaceful", "hopeful",
+  "melancholy", "content", "proud", "grateful",
+  "stressed", "anxious", "loving", "caring",
+  "bored", "curious", "energetic", "sleepy"
+]
```

Comment:
No need to validate moods. It's just limiting potential inputs unnecessarily. String gets passed directly to the spotify query, so no need to worry about it.